modules/imgimport/include/imgimport.h

```diff
@@ -37,6 +37,9 @@
 #include <opencv2/highgui/highgui.hpp>
 #include <string>
 #include <vector>
+#include "csvreading.h"
+
+using namespace std::vector;
```

Comment:
Not quite what I meant. The idea is to avoid "using namespace" entirely in headers and to replace uses of vector with std::vector.